quizapp/src/main/java/billennium/quizapp/service/QuestionService.java

```diff
@@ -74,4 +78,27 @@ public QuestionGetDto getQuestionById(Long id) {
         return questionRepository.findById(id).map(question -> new ModelMapper().map(question, QuestionGetDto.class))
                 .orElse(QuestionGetDto.builder().build());
     }
+
+    public DeleteStatusInfoDto deleteQuestionById(Long id) {
+        Optional<Question> question = questionRepository.findById(id);
```

Comment:
co do samego kodu: trzy razy builder powtarzasz tylko z innymi parametrami, mozna to wydzielic do metody na lajcie i kod sie skrocitrzykrotnie.

Co do samego rozwiazania to nie podoba mi sie. Fajniej wg mnie wygladaloby jeżeli serwer zwracał by dany HTTP Status code. Czyl jak jest usuniete to deleted, a usuniecie pytania ktore ma relacje do quizu nie powinno się udać z punktu widzenia bazy danych -  wiec taka sytuacje obslugujesz lapiac wyjątek i zwracajac jakis HTTP Status typu Conflict czy cos. I takie cos sobie obslugujesz na froncie - nie bedzie musial tutaj definiować zwrotek jak te statiki u gory